### DISCUSSION TITLE
Spurious currents in interReactiveTransferFoam ?

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#33] [Q&A] [answered]
Hi everyone,

I hope you're all doing well!

I understand that GeochemFoam was designed with porous media and microfluidics applications in mind. I'm curious about whether there are any limitations regarding spurious currents for low Capillary numbers in GeochemFoam, specifically in interReactiveTransferFoam ?

Thnaks for the help and time,

Sincerely,
Santhosh

**Comment by Mohamadreza1998** [comment, marked as answer]
Hi Santhosh,

The interReactiveTransferFoam was use continuous species transport (CSF) model and this method have spurious currents for calculation of the surface tension force at small capillary numbers (Ca < 0.01) but the impact of spurious currents are negligible and Dr. Maes and Soulaine have also shown in pervious works that these are negligible compared to the analytical solutions.

If you want more details that can be read Dr. Maes paper (the paragraph below Equation 19).

[https://link.springer.com/article/10.1007/s11242-021-01661-8](url)

**Reply by Santhosh91000** [reply]
Hello Mohamad,

Thanks a lot for the information! This will help me justify the use of the solver :).

Sincerely,
Santhosh
